build/steps-conda.yml

```diff
@@ -7,24 +7,35 @@ steps:
 ##
 # Pre-reqs
 ##
+- task: DownloadBuildArtifacts@0
+  inputs:
+    artifactName: iqsharp
+    downloadPath: '$(System.DefaultWorkingDirectory)/artifact'
+
+- pwsh: |
```

Comment:
how is this going to work with the end-to-end/signed build? name of the drops is different. 
Also, should this be inside a script so we dont' have to copy the same commands?

Reply:
With respect to copying the same commands, my thinking was that it's a single-line script whose parameters vary between the signed and prerelease builds as per your comment, such that making a new script would not save very much over copying and pasting. Happy to discuss that further, though.